axelrod/result_set.py

```diff
@@ -0,0 +1,54 @@
+import numpy
+from StringIO import StringIO
+
+
+class ResultSet(object):
+    """
+    A class to hold the results of a tournament
+    and to return them as plots or csv files.
+    """
+
+    def __init__(self, players, turns, repetitions):
+        self.players = players
+        self.nplayers = len(players)
+        player_list = list(range(len(players)))
+        self.turns = turns
+        self.repetitions = repetitions
+        repetition_list = list(range(repetitions))
+        self.results = [
+            [[0 for irep in repetition_list]
+                for j in player_list] for i in player_list]
+        self.output_initialised = False
+
+    def generate_scores(self):
+        numpy_array_results = numpy.array(self.results)
+        return numpy_array_results.sum(axis=1)
+
+    def generate_ranking(self, scores):
```

Comment:
On all these simple statements, could you add a single line or two of docstrings? Hoping to package this at some point (so people could `pip install axelrod`) so it should throw something up if and when `help` would be run it. Sorry to be a pain. Otherwise: all looks really great.